x/programs/cmd/simulator/testdata/token.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
So the expectation is that users will generate these JSON files in order to test their programs? I haven't seen this pattern in smart contract development done before. Most of the time the framework just deploys the smart contract to a local node and users interact with the contract via a typescript library. I'm not saying this is worse, but certainly different.

Reply:
We will offer both YAML/JSON step scripting and contract-based testing (like Foundry)

Reply:
I used something similar for Rosetta as well: https://github.com/coinbase/rosetta-sdk-go/tree/master/constructor/dsl